### DISCUSSION TITLE
v1.0.0-beta.5

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#7] [Announcements]
[**v1.0.0-beta.5**](https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.0.0-beta.5) has been released. Please leave feedback here and create/update issues as necessary.

/cc @rumpleforeskin83 @Lithradus @dshk0718 @bjongbloed

**Comment by EvanMulawski** [comment]
@bjongbloed Since it is software mode only in this release, would you want the ability to switch between software and hardware modes?

**Reply by bjongbloed** [reply]
Is it possible to leave it how you found it? Assuming you need SW mode to write fan values, if it's in HW mode, put it back in HW mode and otherwise just leave it in SW mode? 

If that's not possible, yes, a setting would be nice so I don't have to run RGB software if I don't want to.

**Reply by EvanMulawski** [reply]
I did some testing for this. Unfortunately, when the device is placed in hardware mode it needs to be put back into software mode in order to communicate with it in any way. This would result in RGB flickering similar to #4 as the device would transition to software mode (in order to read speeds/temperatures and write fan power) and back to hardware mode every second. There is no separation of hardware/software modes for lighting and fan control.
